src/core/utils/compatibility.js

```diff
@@ -31,7 +48,6 @@ export function getCompatibleVersions({ _log = log, addon, clientApp } = {}) {
       _log.info(oneLine`addon is type ${ADDON_TYPE_OPENSEARCH}; no
```

Comment:
Does this message still apply? It seems this `if` block does not do anything anymore.

Reply:
It never really did anything actually. I should remove it. Good call.